src/Commands/Shooter/Shoot.h

```diff
@@ -16,6 +16,8 @@ class Shoot: public Command {
 
 private:
     const float OPTIMAL_RPM = -3300.0f;
+    const float THROTTLE = 500.0f;
     const float IDLE = 1000.0f;
     const float START_MULTIPLIER = 1.25f;
+    const double DEADBAND = 0.01;

```

Comment:
Our deadband values are super inconsistent across the entire codebase. It may be worth making the smooth controller class more robust to easily accommodate different deadband values. For now the only thing we can do is go to competition and see what happens.